.github/workflows/scheduled_builds.yml

```diff
@@ -16,3 +16,5 @@ jobs:
       - run: gh workflow run ci.yml --ref main
       - run: gh workflow run ci.yml --ref 3.1
       - run: gh workflow run ci.yml --ref 3.0
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I am surprised the built in actions token isn't sufficient for this? Do we really need to add a new one?

Reply:
That's what the docs say: https://docs.github.com/en/actions/using-workflows/using-github-cli-in-workflows

Reply:
But there already is a default token on all actions no?

Reply:
Basically I am saying, we need this line, but to do nothing else.

Reply:
You're right! I hadn't realised there was a default token. It has write permissions for Actions on main: https://github.com/sunpy/sunpy/runs/5791457157?check_suite_focus=true#step:1:14 So I shall merge.